extensions/commands/recipe/README.md

```diff
@@ -36,3 +36,73 @@ updating moltenvk/1.2.0 to moltenvk/1.2.2 in conan-center-index/recipes/qt/6.x.x
 updating libjpeg-turbo/2.1.4 to libjpeg-turbo/2.1.5 in conan-center-index/recipes/qt/6.x.x/conanfile.py:386
 Successfully bumped the dependencies of recipe conan-center-index/recipes/qt/6.x.x/conanfile.py
 ```
+
+#### [Create SBOM](cmd_create_sbom.py)
+
+For the graph JSON passed as parameter it creates an SBOM in CycloneDX 1.4 JSON format.
+
+**Parameters**
+- **graph_json**: Path to Conan generated graph JSON output file.
+
+```
+$ conan graph info -f json --requires=openssl/3.1.1 > openssl_graph.json
```

Comment:
> conan graph info -f json --requires=openssl/3.1.1 > openssl_graph.json

better use `cyclonedx:json` as a format.
there might be other json formats in the future.

Reply:
This is a standard command/feature of the client and it's no specific to the SBOM feature being added.

However I do like this suggestion for the line below, having a dedicated `--format` option or having a more specific name for the command 👍

Reply:
I now enforce a format option.

If we allow the usage of external libraries, this would enable us to also output the SBOM in CycloneDX JSON1.2, JSON1.3, JSON1.5, XML1.0, XML1.1, XML1.2, XML1.3, XML1.4, and XML1.5

If I have to drop all external libs and create the output by hand, I will stick to one of them only, probably JSON1.4